app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/composables/ClaimStatusCards.kt

```diff
@@ -15,6 +15,8 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.LocalConfiguration
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import arrow.core.NonEmptyList
```

Comment:
Do you think it would be difficult to roll our own implementation of NonEmptyList?

Reply:
Technically it would not be difficult, but we'd just set ourselves up for more work in the long run by not taking advantage of all the nice things that come out of the box from arrow.
It's not like it's some obscure lib that nobody uses either, it's the most popular Koltin library that does this.
Plus we're not taking in the more specialized arrow components like Arrow Optics that require a kapt implementation too which would slow down builds, we just take core which can be simply seen as an extension to the kotlin stdlib.

Is there some particular reason why you would want to avoid it that you're thinking about? If you want Oscars input too, we did have a discussion some weeks ago where I mentioned looking into arrow core and he said he was positive about looking into it at least.

Reply:
That sounds good. I haven't looked into arrow that much so just wanted to know what its about :) Thanks for the explanation! Go for it! 👍